common/prugpio.h

```diff
@@ -54,11 +54,15 @@
 #define USR1 (1<<22)
 #define USR2 (1<<23)
 #define USR3 (1<<24)
-// The define a couple of GPIO pin addresses
+// The define a couple of GPIO pin addresses on Black
 // GPIO1
 #define P9_14 (1<<18)
 #define P9_16 (1<<19)
 
+// The define a couple of GPIO pin addresses on Pocket
+// GPIO1
+#define P1_32 (1<<10)
```

Comment:
There are a number of pins that boot-up to PRU inputs. I'd guess they all should be defined at some point.

Reply:
Why is this one separate from the others?

Reply:
It's for the Pocket.

Reply:
Yes, the other pins should be in there too.